package.json

```diff
@@ -65,14 +65,10 @@
     "vuex": "4.1.0"
   },
   "peerDependencies": {
-    "@vue/compiler-dom": "^3.0.1",
```

Comment:
Shouldn't `@vue/compiler-dom` still be a peer dependency?

Reply:
Maybe it should, I don't know what's best. As we use it, but people don't necessarily have it installed, I was thinking that only specifying it as a dependency was maybe better.

Reply:
It's a dependency of `vue` so I guess it's installed anyway

Reply:
We can definitely debate what's "techincally" correct but I'd say for 99% of people they will have it installed - how many people are using Vue without compiler-dom?  Seems fine either way.